src/s2_geo.rs

```diff
@@ -0,0 +1,120 @@
+use cgmath::Vector2;
+use nav_types::{ECEF, WGS84};
+use s2::cap::Cap;
+use s2::cellid::CellID;
+use s2::latlng::LatLng;
+use s2::point::Point;
+use s2::rect::Rect;
+use s2::region::RegionCoverer;
+use s2::s1::angle::{Angle, Rad};
+use s2::s1::chordangle::ChordAngle;
+use std::vec::Vec;
+
+const LEVEL_MOD: u8 = 1;
+const EARTH_RADIUS_M: f64 = 6_371_010.0;
+
+fn earth_tangent_m_to_rad(meters: f64) -> f64 {
```

Comment:
I'd be wary of pretending the earth is a sphere. For `cell_ids_radius()` it might be good to have that, but then we should be sure that it overestimates the rad, and it could maybe be made internal to that function.

Reply:
Yes, when actually using it we should have a few more tests... I just wanted to get this in now and concentrate on more important things for now...

Reply:
How about you make it a nested function of `cell_ids_radius()` and optionally add a TODO? Also, I think we get the most `rad` for our `m` at the smallest radius, so let's go with the Arctic Ocean floor for the `EARTH_RADIUS_M` (https://en.wikipedia.org/wiki/Earth_radius#Geophysical_extremes)

Reply:
Best to also hide that constant in `cell_ids_radius()`

Reply:
Oh, I see it's also used in `cell_ids_rect()`. Maybe the best approach is then to suffix these three functions with `_inexact` or `_approx` and add a TODO.

Reply:
How about using the polar radius as defined by WGS84, see https://en.wikipedia.org/wiki/Earth_radius#Global_average_radii and https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84?

Reply:
I actually just changed it to exact computations, PTAL!